### ISSUE TITLE
CPU load high after playing music

### ISSUE THREAD

**Issue by mikk150** [marcelveldt/plugin.audio.spotify#39]
When playing music CPU load rises to about 50%, This is almost expected(encrypted stream? doing whatever it needs to do)

But when I stop the song, CPU load high even if I exit the plugin

```
19:01:46.354 T:3889312752  NOTICE: plugin.audio.spotify --> reported architecture: aarch64
19:01:46.647 T:3889312752  NOTICE: plugin.audio.spotify --> ok spotty v0.6.1
19:01:46.647 T:3889312752  NOTICE: plugin.audio.spotify --> Architecture detected. Using spotty binary /home/osmc/.kodi/addons/plugin.audio.spotify/resources/lib/spotty/arm-linux/spotty-muslhf
19:01:47.934 T:3889312752  NOTICE: plugin.audio.spotify --> ok spotty v0.6.1
19:01:48.173 T:3889312752  NOTICE: plugin.audio.spotify --> started webproxy at port 52308
19:01:51.791 T:3889312752 WARNING: plugin.audio.spotify --> {"accessToken":"<MyAwesoneAccessTokenHere>","expiresIn":3600,"tokenType":"Bearer","scope":["playlist-read-private","playlist-read-collaborative","user-follow-read","playlist-modify-private","user-read-email","user-read-private","user-modify-playback-state","user-follow-modify","user-library-read","user-library-modify","playlist-modify-public","user-read-playback-state","user-read-currently-playing","user-read-birthdate","user-top-read"]}
19:01:51.792 T:3889312752 WARNING: plugin.audio.spotify --> Token from spotty: {'access_token': '<MyAwesoneAccessTokenHere>', 'expires_in': 3600, 'expires_at': 1500051711, 'token_type': 'Bearer', 'scope': 'playlist-read-private playlist-read-collaborative user-follow-read playlist-modify-private user-read-email user-read-private user-modify-playback-state user-follow-modify user-library-read user-library-modify playlist-modify-public user-read-playback-state user-read-currently-playing user-read-birthdate user-top-read', 'refresh_token': '<MyAwesoneAccessTokenHere>'}
19:01:51.792 T:3889312752  NOTICE: plugin.audio.spotify --> Retrieved auth token
19:01:52.543 T:3889312752  NOTICE: plugin.audio.spotify --> Logged in to Spotify - Username: mikk150
19:01:52.544 T:3441681392  NOTICE: plugin.audio.spotify --> Start Spotify Connect Daemon
19:03:16.050 T:3658916848   ERROR: GetDirectory - Error getting
19:03:49.758 T:3563316208   ERROR: Previous line repeats 3 times.
19:03:49.758 T:3563316208 WARNING: plugin.audio.spotify --> start chunked transfer for full track 1IBxtCOTtdU9StLpNK6wZh
```

**Comment by marcelveldt** [comment]
This will be fixed in next update

**Comment by marcelveldt** [comment]
Fixed in v 1.1.34
